lms/djangoapps/ora_staff_grader/mock/urls.py

```diff
@@ -0,0 +1,30 @@
+"""
+URLs for Enhanced Staff Grader (ESG) backend-for-frontend (BFF)
+
+NOTE - This should be the same as ../urls.py
+"""
+
+from django.conf.urls import url
```

Comment:
do we want to update these URLs as well?

Reply:
It was unclear if we'd agreed to the new URLs. If we have, I will update.

Reply:
Updated